### PR TITLE
A few fixes and improvements

### PR DESCRIPTION
**Pull request by AleksMat**
This PR makes a few unrelated changes:

- solves the problem with handling `ffmpeg` package in one of the example notebooks,
  * some updates in dockerfiles according to this issue,
- some code refactoring in `eolearn.geometry.transformations` module,
- a minor documentation fix,
- a minor fix in `fiona` version in requirements,
- a few minor pylint fixes.

Sorry for putting everything into a single PR.

### CODE REVIEW

File: geometry/eolearn/geometry/transformations.py [hunk 1]
```diff
@@ -141,73 +143,69 @@ def _vector_is_timeless(vector_input):
...
 
-        vector_data = self._get_vector_data(eopatch)
+        if self._rasterize_per_timestamp:
+            vector_data = vector_data[vector_data.TIMESTAMP.isin(timestamps)]
 
-        if not timeless:
-            #remove timestamps that are not in the eopatch
-            vector_data = vector_data[vector_data['TIMESTAMP'].isin(eopatch.timestamp)]
+        if self.values_column is not None and self.values is not None:
+            values = [self.values] if isinstance(self.values, (int, float)) else self.values
```
Comment (zigaLuksic) [inline geometry/eolearn/geometry/transformations.py:188]: Why is this needed?

**Review by zigaLuksic — COMMENTED** [review]
I am only slightly concerned about one thing, the rest seems ok :)

File: geometry/eolearn/geometry/transformations.py (hunk 1 above)
Comment (AleksMat) [reply geometry/eolearn/geometry/transformations.py:188]: This `self.values` can either be a single value (i.e. int/float) or a list of values. Perhaps this can already be parsed in init?

Comment (zigaLuksic) [reply geometry/eolearn/geometry/transformations.py:188]: Ah, my bad, i thought it was `values_column` twice, so it was odd that we were checking if its an integer or float.
